test/unit/ngsiv2/contextRequests/ngsildPayloadMeasure2.json

```
@@ -3,7 +3,10 @@
     "type": "AnMQTTDevice",
     "status": {
         "type": "Property",
-        "value": "free"
+        "value": "free",
+        "metadata": {
+            "observedAt":"2012-09-21T12:00:00Z"
+        }
```

This file is for the mock expecations at CB?

This syntax is not correct for metadata. The values for `metadata` sub-keys are objects with `value` and `type`, so it should be:

```suggestion
        "metadata": {
            "observedAt":
                "value": "2012-09-21T12:00:00Z",
                "type": ??
        }
```

What I'm not sure if which `type` to use here... Some candidates: `ngsild`, `ngsildMetadata`, `ngsildTheNameTheyUseForKindOfThingsInTheirSpec`.

---

`propertyOfProperty`?

---

Another possibility is that the JSON type of the value decides, applying the same rules than in CB: https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/orion-api.md#partial-representations

---

What about an empty 'type'? CB will fill an empty type with a proper type ?